config/locales/en.yml

```diff
@@ -632,6 +643,19 @@ en:
       member: Member
       applications: *applications
       applications_tooltip: Number of open applications
+      expire_date: *expire_date
+
+  payments:
+    create:
+      page_title: &page_title '%{payment_type} Payment by: %{user_name}'
+      title: *page_title
+      something_wrong: Something went wrong - contact the system administrator at member@sverigeshundforetagare.se
+
```

Comment:
Shouldn't this message get its e-mail address through a placeholder instead of using a hard-coded one?

Reply:
(What Robert said)  and:  This email address is available in ENV['SHF_MEMBERSHIP_EMAIL'].